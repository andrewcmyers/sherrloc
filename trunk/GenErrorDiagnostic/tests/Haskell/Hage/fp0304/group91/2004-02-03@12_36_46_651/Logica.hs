module Logica where

data Prop
  = En [Prop]
  | Of [Prop]
  | Niet Prop
  | Prop :-> Prop
  | Var String
  | Bool Bool

type Bedeling = [String]



evalueer :: Prop -> Bedeling -> Prop
evalueer p b = head p

-- head p should be: p
-- 16,16-21   16,21-21
