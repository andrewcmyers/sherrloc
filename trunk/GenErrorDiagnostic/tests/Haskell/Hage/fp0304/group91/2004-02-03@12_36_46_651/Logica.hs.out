Compiling Logica.hs
(16,21)-(16,22): Type error in application
 expression       : head p
 function         : head
   type           : [a] -> a
 1st argument     : p
   type           : Prop
   does not match : [a] 

Compilation failed with 1 error
